server/templates/base.html

```diff
@@ -8,6 +8,17 @@
     <link rel="icon" type="image/png" sizes="16x16" href="{% static 'assets/favicon-16x16.png' %}">
     <link rel="icon" type="image/x-icon" href="{% static 'assets/favicon.ico' %}">
     <link rel="mask-icon" href="{% static 'assets/safari-pinned-tab.svg' %}" color="#5bbad5">
+    <!-- Add these meta properties to render domain on Facebook -->
+		<meta property="og:title" content="{{ title }}">
+		<meta property="og:description" content="{{ title }}">
+		<meta property="og:image" content="https://files.gitter.im/iodide-project/iodide/xW1J/iodide-sticker-2.png">
```

Comment:
This should be just `{% static 'assets/apple-touch-icon.png' %}` as above, shouldn't it?